### ISSUE TITLE
The left bar have not change to color of black

### ISSUE THREAD

**Issue by WeiChieh-Chen** [pierrenel/VSOceanDeep#1]
My environment is windows.
![1531407824](https://user-images.githubusercontent.com/16322373/42641791-ee5451e2-8627-11e8-9706-2702321b65ba.png)

**Comment by sendsent** [comment]
Yeah, I  was wondering the same thing with my environment on Mac.
